portfolio/src/main/java/com/google/sps/servlets/StringServlet.java

```diff
@@ -19,9 +19,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //create a new array list and input quoataions.
         ArrayList<String> quotations = new ArrayList<String>();
 
-        quotations.add("<p>The past is not dead. In fact, it's not even past.  -William Faulkner</p>");
-        quotations.add("<p>Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X</p>");
-        quotations.add("<p>Don't count the days, make the days count. -Muhammad Ali</p>");
+        quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
+        quotations.add("Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X");
```

Comment:
"Java code has a column limit of 100 characters." - https://google.github.io/styleguide/javaguide.html#s4.4-column-limit